index.md

```diff
@@ -42,7 +42,7 @@ A monthly test to see of the role still works on the current distributions.
 29. [![Build Status](https://api.travis-ci.org/robertdebock/ansible-role-phpmyadmin.svg?branch=master)](https://travis-ci.org/robertdebock/ansible-role-phpmyadmin) [phpmyadmin](https://galaxy.ansible.com/robertdebock/phpmyadmin/) - [source](https://github.com/robertdebock/ansible-role-phpmyadmin).
 30. [![Build Status](https://api.travis-ci.org/robertdebock/ansible-role-php.svg?branch=master)](https://travis-ci.org/robertdebock/ansible-role-php) [php](https://galaxy.ansible.com/robertdebock/php/) - [source](https://github.com/robertdebock/ansible-role-php).
 31. [![Build Status](https://api.travis-ci.org/robertdebock/ansible-role-postfix.svg?branch=master)](https://travis-ci.org/robertdebock/ansible-role-postfix) [postfix](https://galaxy.ansible.com/robertdebock/postfix/) - [source](https://github.com/robertdebock/ansible-role-postfix).
-32. [![Build Status](https://api.travis-ci.org/robertdebock/ansible-role-python_pip.svg?branch=master)](https://travis-ci.org/robertdebock/ansible-role-python_pip) [python_pip](https://galaxy.ansible.com/robertdebock/python_pip/) - [source](https://github.com/robertdebock/ansible-role-python_pip).
+32. [![Build Status](https://api.travis-ci.org/robertdebock/ansible-role-python-pip.svg?branch=master)](https://travis-ci.org/robertdebock/ansible-role-python_pip) [python_pip](https://galaxy.ansible.com/robertdebock/python-pip/) - [source](https://github.com/robertdebock/ansible-role-python_pip).
```

Comment:
Thanks Dalibor, sure I remember you, good to "see" you again!

Ansible Galaxy recently changed the syntax for role-names, people could use dashes (-) before, but since a recent release, this has been changed to underscores (_). I'll fix this line now, but your merge request also refers to a non-existent URL. (https://galaxy.ansible.com/robertdebock/python-pip)

CU Dalibor, thanks for your contribution!